ibc/relayer_test.go

```diff
@@ -0,0 +1,34 @@
+package ibc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestChannelOptsConfigured(t *testing.T) {
+	// Test the default channel opts
+	opts := DefaultChannelOpts()
+	require.True(t, opts.IsFullyConfigured())
+
+	// Test empty struct channel opts
+	opts = CreateChannelOptions{}
+	require.False(t, opts.IsFullyConfigured())
+
+	// Test invalid Order type in channel opts
+	opts = CreateChannelOptions{
+		SourcePortName: "transfer",
+		DestPortName:   "transfer",
+		Order:          3,
+		Version:        "123",
+	}
+	require.False(t, opts.IsFullyConfigured())
```

Comment:
Seems like this should be a different validation. The order is configured, but it has an invalid value.

Reply:
I've changed it up a bit locally, just trying to get tests passing and i'll have another commit here that addresses this.